keyboards/irene/timer1.h

```diff
@@ -0,0 +1,10 @@
+#ifndef TIMER1
+#define TIMER1
+#include <stdio.h>
+#include <avr/io.h>
+#include <util/delay.h>
+
+void timer1_init(void);
+void timer1_stop(void);
```

Comment:
why are we passing void?

Reply:
I'm passing void to specify that the function takes no input argument. Removing void results in "error: function declaration is not a prototype."